advanced_functionality/autogluon-tabular/container-training/train.py

```diff
@@ -128,15 +136,16 @@ def train(args):
         pickle.dump(column_dict, f)
     
     # Train models
-    predictor = task.fit(
-        train_data=train_data,
-        output_directory=args.model_dir,
-        **args.fit_args,
-    )
+
+    args.fit_args.pop('label', None)
```

Comment:
perhaps also let users specify `eval_metric, sample_weight, weight_evaluation, learner_kwargs`? These are specified in init, goes in same place as label. ([doc link](https://auto.gluon.ai/stable/_modules/autogluon/tabular/predictor/predictor.html#TabularPredictor))

Perhaps we can call them `args.init_args`?

Reply:
This also means `label` can be moved to `init_args`, or made as its own things similar to `args.model_dir`

Reply:
yes, you could also maybe just have model_dir be in init_args as `path` and remove `model_dir`, but unsure the reasoning behind current design.

Reply:
`model_dir` is ENV VAR off of sagemaker. It is safe to do it this way.